Sources/MapboxMaps/Annotations/ViewAnnotation.swift

```diff
@@ -0,0 +1,117 @@
+import UIKit
+@_implementationOnly import MapboxCommon_Private
+@_implementationOnly import MapboxCoreMaps_Private
+
+public protocol ViewAnnotationManager: AnyObject {
+    // TODO: Add documentation
+    func addViewAnnotation(_ annotatonView: UIView, _ options: ViewAnnotationOptions)
```

Comment:
in order not to forget from our discussion today: rename param here and below to `view` for consistency with Android and in general it seems to fit better.